### PR TITLE
MiBeacon: add motion sensor support

### PR DESCRIPTION
**Pull request by unrelentingtech**

### CODE REVIEW

File: main/broadcasters.c [hunk 1]
```diff
@@ -548,6 +561,25 @@ static void mijia_sensor_metadata_get(uint8_t *adv_data, size_t adv_data_len,
...
+            }
+            else if (device_type == MIJIA_DEVICE_TYPE_MJYD02YL)
+            {
+                cb("Light", val == 100 ? "1" : "0", ctx);
+            }
+            else if (device_type == MIJIA_DEVICE_TYPE_RTCGQ02LM)
+            {
+                cb("Light", val == 256 ? "1" : "0", ctx);
+            }
+            cb("Motion", "1", ctx);
```
Comment (shmuelzon) [inline main/broadcasters.c:581]: So, if the data type is `MIJIA_SENSOR_DATA_TYPE_MOTION_LIGHT` then motion is always detect? How is motion reset? Will the new message be with data type `MIJIA_SENSOR_DATA_TYPE_MOTION` from the same device?

Comment (unrelentingtech) [reply main/broadcasters.c:581]: No, there is just no reset timer on the device. You can only reset to zero inside your home automation software, e.g. with the "expire" attribute in openHAB.

File: main/broadcasters.c [hunk 2]
```diff
@@ -548,6 +561,25 @@ static void mijia_sensor_metadata_get(uint8_t *adv_data, size_t adv_data_len,
...
+                cb("Illuminance", s, ctx);
+            }
+            else if (device_type == MIJIA_DEVICE_TYPE_MJYD02YL)
+            {
+                cb("Light", val == 100 ? "1" : "0", ctx);
+            }
+            else if (device_type == MIJIA_DEVICE_TYPE_RTCGQ02LM)
+            {
+                cb("Light", val == 256 ? "1" : "0", ctx);
+            }
```
Comment (shmuelzon) [inline main/broadcasters.c:580]: Coding style nitpicking: Please remove the curly brackets from single line blocks (unless the statement above it is multi line).

**Review by shmuelzon — CHANGES_REQUESTED** [review]

File: main/broadcasters.c (hunk 1 above)
Comment (unrelentingtech) [reply main/broadcasters.c:581]: At least on the RTCGQ02LM the light is also broadcast separately as a `MIJIA_SENSOR_DATA_TYPE_LIGHT` message

> publishing motion twice, in succession, 0 and then 1

Not necessary IMO

> perhaps we should publish `true`/`false`

ehh, there's plenty of other types that already report a byte directly that are 0/1, let's not introduce inconsistencies

Comment (shmuelzon) [reply main/broadcasters.c:581]: OK, I'm with you

File: main/broadcasters.c (hunk 2 above)
Comment (shmuelzon) [reply main/broadcasters.c:580]: Please just fix this and I'll merge

Comment (unrelentingtech) [reply main/broadcasters.c]: done

**Review by shmuelzon — APPROVED** [review]

**Comment by shmuelzon** [comment]
Thanks!

**Pull request merged**